splinterd/src/node/runnable/biome.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Builder for configuring Biome REST API resources
+
+use std::sync::Arc;
+
+use splinter::biome::{
+    credentials::rest_api::BiomeCredentialsRestResourceProviderBuilder,
+    key_management::rest_api::BiomeKeyManagementRestResourceProvider,
+    profile::rest_api::BiomeProfileRestResourceProvider,
+};
+use splinter::error::InternalError;
+use splinter::rest_api::{
+    actix_web_1::Resource as Actix1Resource, AuthConfig, RestResourceProvider,
+};
+use splinter::store::StoreFactory;
+
+/// Biome resource provider
+pub struct BiomeResourceProvider {
+    /// Biome's ActixWeb1 resources
+    pub(crate) actix1_resources: Vec<Actix1Resource>,
+    /// Biome-specific authorization configurations
+    pub(crate) auth_configs: Vec<AuthConfig>,
+}
+
+impl BiomeResourceProvider {
+    /// Take the available REST Resources from the Biome resource provider.
+    pub fn take_actix1_resources(&mut self) -> Vec<Actix1Resource> {
+        let mut replaced = vec![];
+        std::mem::swap(&mut self.actix1_resources, &mut replaced);
+        replaced
+    }
+
+    pub fn create_biome_actix1_resources(
```

Comment:
Suggest renaming this to `new`, as it returns a provider, not biome_actix1_resources

Reply:
Updated this method to just `new`